pkg/kubefed2/util/util.go

```diff
@@ -96,8 +96,8 @@ func ClusterClientset(config *rest.Config) (*kubeclient.Clientset, error) {
 
 // ClusterRegistryClientset provides a cluster registry API compliant
 // clientset to communicate with the cluster registry.
-func ClusterRegistryClientset(config *rest.Config) (*crclient.Clientset, error) {
-	return crclient.NewForConfig(config)
+func ClusterRegistryClientset(config *rest.Config) (generic.Client, error) {
```

Comment:
(No action required) What's the advantage of maintaining this method vs just replacing it with calls to `generic.New`?

Reply:
No advantage, just that code change was big. It can be taken up as next step in simplifying the code.